app/whats-new/views/content.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layouts/base.njk" %}
+
+{% set heading = "What's New" %}
+
+{% block pageTitle %}
+  {{ heading }} – {{ super() }}
+{% endblock %}
+
+{% block content %}  
+  {% if contentBody and contentTitle %}
+    <header>
+      <h1 class="govuk-heading-xl">
+        {{ heading }}
+      </h1>
+    </header>
+    
+    <h2 class="govuk-heading-l">
+      {{ contentTitle }}
+    </h2>
+    
+    {{ contentBody | safe }}
+    {% else %}
+    
+    {% endif %}
+{% endblock %}
```

Comment:
It looks like there's a newline missing at the end of this file, it's flagged by GitHub as typically there would be one.

Reply:
Good spot